vmdb/app/controllers/api_controller/vms.rb

```diff
@@ -92,5 +106,16 @@ def destroy_vm(vm)
     rescue => err
       action_result(false, err.to_s)
     end
+
+    def set_owner_vm(vm, owner)
+      desc = "#{vm_ident(vm)} setting owner to '#{owner}'"
+      user = User.find_or_create_by_ldap_upn(owner)
```

Comment:
I hadn't merged this because I meant to comment on this part, but I guess I didn't.   Is this supposed to be hard linked to ldap?

Also I thought setting the owner automatically set the group.   @lfu?